com.ibm.streamsx.json/impl/java/src/com/ibm/streamsx/json/converters/JSONToTupleConverter.java

```diff
@@ -0,0 +1,404 @@
+package com.ibm.streamsx.json.converters;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+import com.ibm.json.java.JSONArray;
+import com.ibm.json.java.JSONObject;
+import com.ibm.streams.operator.Attribute;
+import com.ibm.streams.operator.StreamSchema;
+import com.ibm.streams.operator.Tuple;
+import com.ibm.streams.operator.Type;
+import com.ibm.streams.operator.logging.TraceLevel;
+import com.ibm.streams.operator.meta.CollectionType;
+import com.ibm.streams.operator.meta.TupleType;
+import com.ibm.streams.operator.types.RString;
+import com.ibm.streams.operator.types.Timestamp;
+
+public class JSONToTupleConverter {
+
+	private static Logger l = Logger.getLogger(JSONToTupleConverter.class.getCanonicalName());
+	
+	/**
+	 * Convert a JSON object to a Java object of the specified type. 
```

Comment:
It's a  "JSON value" , not a "JSON object", JSON object would typically be assumed to be something that was "{ ... }".

In general you don't really need to use "object" or "Java"  to described returned values in Javadoc, pretty much everything is a Java object so that's assumed, or can be inferred from the type not being a primitive. Try to describe parameters and return values logically. So the return is something like:

`@return Value converted to SPL representation with type {@code type}